### PR TITLE
add note about documentation being out of sync

### PR DESCRIPTION
**Pull request by arve0**
ref https://github.com/actix/actix-web/issues/901

image

### CODE REVIEW

**Comment by krircc** [comment]
There is on going work for this. https://github.com/actix/actix-website/pull/90

**Comment by arve0** [comment]
@krircc Indeed, this is just a warning until the new documentation gets merged.

**Comment by cldershem** [comment]
@arve0 Thanks for doing this. It's taking me a bit longer than I had expected to update all of the docs. In order for curious users to get a full picture of how to use all of this I think the [examples](github.com/actix/examples) should be linked to also. Maybe reword it to be something like:

"Looking for actix-web 1.0 docs? This book is currently [out of date](#90). In the meantime checkout the [API docs]() and the [examples repo]()."

**Comment by arve0** [comment]
Feel free to do any adjustments [directly to the PR](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Comment by akappel** [comment]
Oh yes, please merge this asap. I was definitely caught spinning my wheels because of the current transition from 0.7 to 1.0. It was confusing and frustrating to say the least. Looking forward to the complete overhaul, @cldershem!

File: layouts/partials/header.html [hunk 1]
```diff
@@ -54,4 +54,12 @@
           </ul>
         </div>
       </nav>
+      <div id="heads-up" class="alert alert-info" role="alert">
+        Looking for actix-web 1.0 docs? This book is currently
+        <a href="https://github.com/actix/actix-website/pull/90" class="alert-link">out of date</a>.
+        In the meantime checkout the
+        <a href="https://docs.rs/actix-web/1.0.2/actix_web/" class="alert-link">API docs</a>
```
Comment (cldershem) [inline layouts/partials/header.html]: Change this link to "https://docs.rs/actix-web/latestactix_web/". I just noticed there was a release in the last day.

Comment (arve0) [reply layouts/partials/header.html]: Done.
